modules/installation-vsphere-machines.adoc

```diff
@@ -12,11 +12,26 @@ Before you install a cluster that contains user-provisioned infrastructure on VM
 .Prerequisites
 
 * Obtain the Ignition config files for your cluster.
-* Have access to an HTTP server that you can access from your computer and that the machines that you create can access.
 * Create a link:https://docs.vmware.com/en/VMware-vSphere/6.0/com.vmware.vsphere.vcenterhost.doc/GUID-B1018F28-3F14-4DFE-9B4B-F48BBDB72C10.html[vSphere cluster].
 
 .Procedure
 
+. Locate the bootstrap Ignition config file `<installation_directory>/bootstrap.ign` that the installation program created.
+
+. Store the Ignition config file in the extra configuration properties of the virtual machine (VM). For example:
```

Comment:
@bobfuru I am unsure this is useful to add a snip of terraform. It can't be executed as-is and since terraform isn't listed might confuse the reader what we are trying convey.

Reply:
Ack. Removed the terraform sample.